### PR TITLE
docs: update status for CIP-6, CIP-17, CIP-20

### PR DESCRIPTION
**Pull request by rootulp**
The CIPs that are prerequisites to Lemongrass (CIP-17) have all been moved to final so now CIP-17 can move from draft to review or last call.

### CODE REVIEW

File: cips/cip-17.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ title: Lemongrass Hardfork
 description: Reference specifications included in the Lemongrass Hardfork
 author: Evan Forbes (@evan-forbes)
 discussions-to: https://forum.celestia.org/t/lemongrass-hardfork/1589
-status: Draft
```
Comment (rootulp) [inline cips/cip-17.md:7]: is it possible to move from draft -> last call? or should I move from draft -> review for some time (i.e. a week) -> last call with a 2 week deadline?

Comment (jcstein) [reply cips/cip-17.md:7]: it isn't abundantly clear, but it is clear enough that it probably _should_ follow each step https://github.com/celestiaorg/CIPs/blob/main/cips/cip-1.md#cip-process

File: cips/cip-17.md [hunk 2]
```diff
@@ -4,7 +4,8 @@ title: Lemongrass Hardfork
...
+status: Last Call
```
Comment (jcstein) [inline cips/cip-17.md:7]:
```suggestion
status: Review
```

File: cips/cip-17.md [hunk 3]
```diff
@@ -4,7 +4,8 @@ title: Lemongrass Hardfork
...
+last-call-deadline: 2024-06-05
```
Comment (jcstein) [inline cips/cip-17.md:8]:
```suggestion
```

File: cips/cip-17.md (hunk 1 above)
Comment (rootulp) [reply cips/cip-17.md:7]: In the CIP call today, Yaz said it was fine to go straight to last call

Comment (jcstein) [reply cips/cip-17.md:7]: sgtm

**Review by jcstein — APPROVED** [review]

**Pull request merged**